metacat-common/src/main/java/com/netflix/metacat/common/json/MetacatJsonException.java

```diff
@@ -17,10 +17,28 @@
  */
 package com.netflix.metacat.common.json;
 
+import lombok.Getter;
+import lombok.Setter;
+
 /**
  * Metacat JSON utility related exception.
  */
 public class MetacatJsonException extends RuntimeException {
+    @Getter
```

Comment:
Annotations should be on the class that way new members get it also. Also this should be immutable so there's no reason to have a setter.